### DISCUSSION TITLE
How to change the current-dir to source-dir when CMake-Tool do configure?

### DISCUSSION THREAD

**Discussion by buyuer** [microsoft/vscode-cmake-tools#2853] [General]
I want that running the 'pwd' command can return the source-dir in CMakeLists.txt when CMake-Tool do configure. I notice that the current-dir is the path where 'nodojs' be, instead of the soruce-dir. So how can I to change the current-dir, such as run ‘cd source-dir’ before CMake-Tools run cmake. Thanks brothers

**Comment by elahehrashedi** [comment]
The only way to do this would be to define a configure task and set the "cwd" option,
```
	"options": {
		"cwd": "path"
	},
```
Here is the link to the tasks documentation: https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/tasks.md
